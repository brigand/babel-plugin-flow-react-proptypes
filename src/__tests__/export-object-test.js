const babel = require('babel-core');
const content = `
export type Foo = {
  a_string: string,
};

console.log('test');
`;

it('export-object', () => {
  const res = babel.transform(content, {
    babelrc: false,
    presets: ['env', 'stage-1', 'react'],
    plugins: ['syntax-flow', require('../')],
  }).code;
  expect(res).toMatchSnapshot();
});
