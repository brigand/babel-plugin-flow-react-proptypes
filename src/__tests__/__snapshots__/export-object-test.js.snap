// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`export-object 1`] = `
"'use strict';

var babelPluginFlowReactPropTypes_proptype_Foo = {
  a_string: require('prop-types').string.isRequired
};
if (typeof exports !== 'undefined') Object.defineProperty(exports, 'babelPluginFlowReactPropTypes_proptype_Foo', {
  value: babelPluginFlowReactPropTypes_proptype_Foo,
  configurable: true
});


console.log('test');"
`;
